provision/testdata/base_case.apic.txt

```diff
@@ -775,17 +775,17 @@ None
                     },
                     "children": [
                         {
-                            "fvSubnet": {
+                            "fvRsCtx": {
                                 "attributes": {
-                                    "ip": "10.2.0.1/16",
+                                    "tnFvCtxName": "kube",
                                     "annotation": "orchestrator:aci-containers-controller"
                                 }
                             }
                         },
                         {
-                            "fvRsCtx": {
+                            "fvRsBDToOut": {
```

Comment:
This doesn't seem right because now we have completely lost the fvSubnet. Similarly in test cases below too.

Reply:
Just checked of the total 30 apic.txt files in testdata the below mentioned 20 names are the ones without fvSubnet - 

base_casea.apic.txt
base_casea_ipv6.apic.txt
flavor_dockerucp.apic.txt
flavor_openshift_310.apic.txt
flavor_openshift_311.apic.txt
nested-elag.apic.txt
nested-portgroup.apic.txt
nested-vlan.apic.txt
nested-vxlan.apic.txt
vlan_case.apix.txt
with_comments.apic.txt
with_interface_mtu.apic.txt
with_new_naming_convention.apic.txt
with_new_naming_convention_dockerucp.apic.txt
with_new_naming_convention_openshift.apic.txt
with_overrides.apic.txt
with_preexisting_tenant.apic.txt
with_tenant_l3out.apic.txt
with_wait_for_network.apic.txt

so in children of fvBD with kube-pod-bd as attribute I guess I need to add this chunk of code after comma in all these remaining 20 files of apic.txt - 
,
{
    "fvSubnet": {
        "attributes": {
            "ip": "10.2.0.1/16",
            "scope": "public",
            "annotation": "orchestrator:aci-containers-controller"
        }
    }
}

LGTM now I guess but does it sounds good to you, Sumit?